packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -78,4 +81,162 @@ describe('test project paths', () => {
       expect(relativeToolsFolder).toEqual(path.join(FAKE_STATE_FOLDER, TOOLS));
     });
   });
+
+  describe('test toolsFolder', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.toolsFolder).toBeDefined();
+    });
+
+    it('should return a path to the tools folder based on root workspace', () => {
+      const toolsFolder = projectPaths.toolsFolder();
+      expect(toolsFolder).toEqual(path.join(FAKE_WORKSPACE, TOOLS));
+    });
+  });
+
+  describe('test metadataFolder', () => {
+    jest.setTimeout(10000);
```

Comment:
why is this required?

Reply:
Tests were failing without it, but now apparently it's been fixed automagically 🤡